utils.c

```diff
@@ -558,3 +558,58 @@ int janus_pidfile_remove(void) {
 	g_free(pidfile);
 	return 0;
 }
+
+void janus_get_json_type_name(int jtype, gboolean positive_non_empty, char *type_name) {
+	if(positive_non_empty)
```

Comment:
Can you add brackets ```{...}``` to this? It makes it less ambiguous, since the action is on multiple lines.